src/utils/socketHooks.js

```diff
@@ -0,0 +1,49 @@
+import {
+  createElement,
+  createContext,
+  useState,
+  useEffect,
+  useContext
+} from 'react'
+import { Socket } from 'phoenix'
+import { useUserSessions } from '../pages/Account/SettingsSessions'
+import { useUser } from '../stores/user'
+
+export const HOST_NAME = new URL(process.env.REACT_APP_BACKEND_URL).hostname
```

Comment:
`process.env.REACT_APP_BACKEND_URL` is not available during deploy. Use the `window.location.hostname`